### DISCUSSION TITLE
Horizontal split

### DISCUSSION THREAD

**Discussion by domust** [elementary/triage#425] [Q&A]
Hi, I've physically rotated my monitor clockwise and updated display rotation via Display section of the System Settings so that I could fit more code into the screen when reading, which works really well. The problem I've encountered is that now I have enough screen real estate to do horizontal instead of vertical splits, but I can't seem to figure out how to do it. Is this supported?

**Comment by lenemter** [comment]
Duplicate of https://github.com/elementary/gala/issues/380
